tasks/libs/releasing/json.py

```diff
@@ -41,6 +43,12 @@
     "datadog-agent-macos-build": "master",
     "datadog-agent": "main",
 }
+DEFAULT_BRANCHES_AGENT6 = {
+    "omnibus-software": "6.53.x",
+    "omnibus-ruby": "6.53.x",
+    "datadog-agent-macos-build": "6.53.x",
```

Comment:
This branch doesn't exist (as we don't build the macos-agent in Agent6). We might need to create it for compatibility of `release` tasks